7.1-alpine/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM php:7.1-fpm-alpine
 
 ENV COMPOSER_ALLOW_SUPERUSER 1
 ARG ASUSER
+ARG ENABLE_XDEBUG
```

Comment:
although it seems clear enough, undoubtedly clearer would be `ARG ENABLE_XDEBUG "false"` or `0`